detox/scripts/postinstall.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh
```

Comment:
hey @jhen0409 
Is there a reason you removed the `-e` flag ?

Reply:
It may unnecessary for the current case, but I'll add back it to be consistent. 👍